### ISSUE TITLE
Task with focus: true does not focus panel if panel already showed

### ISSUE THREAD

**Issue by inwardmovement** [microsoft/vscode#129796]
A task with:

```
"presentation": {
  "reveal": "always",
  "panel": "dedicated",
  "clear": true,
  "focus": true
}
```

will not focus the panel if it was already showed.
It will focus the panel only after revealing it if it was hidden.

Does this issue occur when all extensions are disabled?: Yes

```
Version: 1.58.2 (user setup)
Commit: c3f126316369cd610563c75b1b1725e0679adfb3
Date: 2021-07-14T22:10:15.214Z
Electron: 12.0.13
Chrome: 89.0.4389.128
Node.js: 14.16.0
V8: 8.9.255.25-electron.0
OS: Windows_NT x64 10.0.19042
```

Steps to Reproduce:
1. Launch task with panel already showed.

**Comment by alexr00** [comment]
duplicate of https://github.com/microsoft/vscode/issues/124667
